AdminWebsite/AdminWebsite/ClientApp/src/app/booking/breadcrumb/breadcrumbItem.model.ts

```diff
@@ -1,14 +1,16 @@
 export class BreadcrumbItemModel {
```

Comment:
I know this isn't your change, so no need to do anything but for future reference: this class can just be an interface. The ctor is pointless.